.goreleaser.yml

```diff
@@ -15,6 +15,8 @@ builds:
       - arm64
       - arm
       - 386
+    ldflags:
+      - -s -w -X go.mercari.io/yo/cmd.version={{.Version}}
```

Comment:
Default is `-s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser`.

https://goreleaser.com/customization/build/